### PR TITLE
Add terms page and content

### PR DESCRIPTION
**Pull request by jenlu33**
Asana ticket: [https://app.asana.com/0/1203289004376659/1204452198889973/f](https://app.asana.com/0/1203289004376659/1204452198889973/f)

Screenshot 2023-05-18 at 1 12 04 PM

### CODE REVIEW

**Review by sarahraqueld — APPROVED** [review]
Looks good!

File: app/views/pages/terms.html.erb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<%= render partial: "occupation_standards/header" %>
```
[the remark this answers is not in the document]
Comment (jenlu33) [reply app/views/pages/terms.html.erb:1]: True, I don't think we need it, but is there a way to have the header without the search bar?

[the remark this answers is not in the document]
Comment (littleforest) [reply app/views/pages/terms.html.erb:1]: Ohhh, I didn't realize those were tied up together. Can you just wrap it in a conditional for now? Exclude if `current_page?(terms_path)` (or whatever the route is). I'll make a ticket to separate those out.

File: app/views/occupation_standards/_header.html.erb [hunk 2]
```diff
@@ -6,12 +6,14 @@
     <!-- <h1 class="page-title" mb-3 text-3xl font-bold leading-none tracking-tight text-gray-100 md:text-4xl xl:text-5xl">Page Title</h1> -->
 
     <! -- If Occupation Standards Page -->
+  <% if current_page?(home_page_path) %>
```
Comment (littleforest) [inline app/views/occupation_standards/_header.html.erb:9]: @jenlu33 this makes us lose the search form on the occupation standards page. Can we do:

```
<% unless current_page?(terms_page_path) %>
```

instead?

**Pull request merged**